CMakeLists.txt

```diff
@@ -4,28 +4,32 @@
 cmake_minimum_required (VERSION 3.6)
 
 # Set build type variable
-#	so RelWithDebInfo is treated as Release
+# so RelWithDebInfo is treated as Release
 if (${CMAKE_BUILD_TYPE} MATCHES "Debug")
-	set(AD_BUILD_TYPE "Debug")
+    set(AD_BUILD_TYPE "Debug")
 else ()
-	set(AD_BUILD_TYPE "Release")
+    set(AD_BUILD_TYPE "Release")
 endif ()
 
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY $<1:>)
```

Comment:
Add comment here, linking to CMake docs as in pr.

Reply:
NB this PR is closed.  It has been migrated to https://github.com/awf/ADBench/pull/39